apps/site/layouts/layouts.module.css

```diff
@@ -228,9 +228,8 @@
 
     main {
       @apply max-w-[660px]
-        gap-4
-        break-all
-        lg:break-normal;
+        gap-4;
```

Comment:
```suggestion
        gap-4
```

no semicolon needed here